src/modules/sdk.js

```diff
@@ -0,0 +1,104 @@
+const assert = require('assert');
+const AWS = require('aws-sdk-wrap');
+const objectPaths = require('obj-paths');
+
+
+class DefaultEntryNotFoundError extends Error {
+  constructor() {
+    super('Entry not found.');
+  }
+}
+class DefaultEntryExistsError extends Error {
+  constructor() {
+    super('Entry exists.');
+  }
+}
+const DefaultEntryNotFound = ({ id }) => new DefaultEntryNotFoundError(id);
+const DefaultEntryExists = ({ id }) => new DefaultEntryExistsError(id);
+
+module.exports = ({
+  name,
```

Comment:
```suggestion
  modelName,
```

Reply:
I think name is good, since this is now called model